plugins/formal-verification/formver.cli/src/org/jetbrains/kotlin/formver/FormalVerificationCommandLineProcessor.kt

```diff
@@ -37,6 +48,11 @@ class FormalVerificationCommandLineProcessor : CommandLineProcessor {
                 "full_viper_dump" -> configuration.put(LOG_LEVEL, LogLevel.FULL_VIPER_DUMP)
                 else -> throw CliOptionProcessingException("Invalid setting $value for ${option.optionName}")
             }
+            UNSUPPORTED_FEATURE_BEHAVIOUR_OPTION -> when (value) {
```

Comment:
Can we parse the `value` String in an `enum`, and then we check its value?

Reply:
I don't think this would make things much simpler, these arguments are typically lowercase anyway.